Automattic/jetpack
# Video block: move Description from beta to production

Move chapters feature from beta to production.

### Proposed changes
Show the Details section on the extended video block for all users, and not only users with beta features enabled.
<img width="438" alt="Screenshot 2023-03-02 at 20 12 24" src="https://user-images.githubusercontent.com/16329583/222529471-5421579b-670a-4e6a-9d14-d84e5f9e5d40.png">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this change to your sandbox
* Go to a Simple dotcom site
* Go to the editor and verify that the section is visible when unproxied
* Confirm that it shows when proxied and it does not have the `beta` flag

## To do:
- [x] Move description from beta to production
- [x] Update the chapters generation in v5 to reflect changes made on v6 to flag the chapters file as auto-generated
- [ ] Change how we are detecting if a transformation can be done, as that is not working when beta is disabled even though v6 is registered
